test/quantization/test_quantize_script.py

```diff
@@ -1868,11 +1867,12 @@ def forward(self, x):
 
         data = [(torch.rand((1, 3, 10, 10), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
         for instance in [BNRelu(True), BNRelu(False), BNFuncRelu(), BNFuncInplaceRelu()]:
-            model = self._test_op_impl(instance, data, "quantized::batch_norm2d_relu")
-            FileCheck().check_not("aten::batch_norm") \
-                       .check_not("aten::relu") \
-                       .check_not("aten::relu_") \
-                       .run(model.graph)
+            for tracing in [True, False]:
```

Comment:
This test: test_qbatch_norm_relu can only run in FBGEMM as batch norm is not supported as an op in qnnpack, please add a check for qengine for this test.

Reply:
OK, I'll do skipIfNoFBGEMM